docker/content-build/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM circleci/python:3.7.5-buster-node
+FROM circleci/3.8.6-buster-node-browsers
```

The version you put in seems wrong. It should be:
```suggestion
FROM circleci/python:3.8.6-buster-node
```

---

Right